sympy/integrals/integrals.py

```diff
@@ -1047,23 +1047,29 @@ def _eval_as_leading_term(self, x):
                 break
         return integrate(leading_term, *self.args[1:])
 
-    def as_sum(self, n, method="midpoint"):
+    def as_sum(self, n, method="midpoint", evaluate=True):
```

Comment:
How about using `Symbol('n', integer=True, positive=True)` as default `n` if `n` is not given?

Reply:
Yes, it makes sense. But I guess it should be a dummy if a symbol isn't provided, to avoid clashing with any `n` in user's context.